examples/parallel/distributed_backend_simple.py

```diff
@@ -0,0 +1,57 @@
+"""
+Using distributed for single_machine parallel computing
+========================================================
+
+This example shows the simplest usage of the dask `distributed
+<https://distributed.readthedocs.io>`__ backend, on the local computer.
+
+This is useful for prototyping a solution, to later be run on a truly
+distributed cluster, as the only change to be made is the address of the
+scheduler.
+
+Another realistic usage scenario: combining dask code with joblib code,
+for instance using dask for preprocessing data, and scikit-learn for
+machine learning. In such a setting, it may be interesting to use
+distributed as a backend scheduler for both dask and joblib, to
+orchestrate well the computation.
+
+"""
+
+###############################################################################
+# Setup the distributed client
+###############################################################################
+from distributed import Client
+# Typically, to execute on a remote machine, the address of the scheduler
+# would go there
+client = Client()
+
+# Recover the address
+address = client.scheduler_info()['address']
+
+# This import registers the dask backend for joblib
```

Comment:
import**s** and registers

Reply:
s/dask/dask.distributed/

Reply:
I think that that was correct: import is used as a noun here.